examples/amp-by-example/amp-by-example.html

```diff
@@ -1260,26 +1260,8 @@ <h3 class="mb1">
           <small>AMP by Example is a part of Accelarated Mobile Pages Project</small>
       </footer>
       <!-- End Footer -->
-        <amp-analytics type="googleanalytics" id="analytics">
-      <script type="application/json">
-        {
-          "vars": {
-            "account": "UA-73836974-1"
-          },
-          "triggers": {
-            "default pageview": {
-              "on": "visible",
-              "request": "pageview",
-              "vars": {
-                "title": "AMP by Example"
-              }
-            }
-          }
-        }
-      </script>
-    </amp-analytics>
     <amp-install-serviceworker
-      src="/sw.js"
+      src="/dist/sw.max.js"
```

Comment:
These are very different things. `sw.max.js` is the Cache Service worker, which is never installed by `<amp-install-serviceworker>`